### ISSUE TITLE
Save the attackers function in the config

### ISSUE THREAD

**Issue by SkyViewBlobio** [calamity-inc/Stand-Feedback#7]
Hi, im gonna make it short: basically if we want to troll someone we have to manually predefine what NPC model the attacker will be, what car they are spawning in, what guns they have, if the cars should be filled or if they should be invulnerable or not. 

Doing this for every new player is a bit tedious especially if stand has multiple other convenient options. 

Please consider adding that

**Comment by rtherhsfhsfhfsdh** [comment]
I agree. Kind of like a "preset tab" on spawning attackers (in which you can save your own version). Would also be cool to be able to search, or even manually imput attacker model names directly instead of scrolling for a while...
